src/app/_client-services/next_api_client_service.ts

```
@@ -1083,4 +1088,19 @@ export class NextApiClientService {
 		const { url, method } = await this.getRouteConfig({ route: EApiRoute.GET_USER_POSTS, routeSegments: [address, 'posts'], queryParams });
 		return this.nextApiClientFetch<IUserPosts>({ url, method });
 	}
+
+	static async addPollVote({ proposalType, index, pollId, decision }: { proposalType: EProposalType; index: string; pollId: string; decision: string }) {
+		const { url, method } = await this.getRouteConfig({ route: EApiRoute.ADD_POLL_VOTE, routeSegments: [proposalType, index, 'poll', pollId, 'votes'] });
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)